packages/frontend-core/src/components/grid/stores/clipboard.ts

```diff
@@ -1,10 +1,41 @@
-import { derived, writable, get } from "svelte/store"
+import { derived, writable, get, Writable, Readable } from "svelte/store"
 import { Helpers } from "@budibase/bbui"
 import { parseCellID, getCellID } from "../lib/utils"
 import { NewRowID } from "../lib/constants"
+import { Store as StoreContext } from "."
 
-export const createStores = () => {
-  const clipboard = writable({
+type ClipboardStoreData =
+  | {
+      value: string[][]
+      multiCellCopy: true
+    }
+  | {
+      value: string | null | undefined
+      multiCellCopy: false
+    }
```

Comment:
This is really cool - I didn't know you can basically conditionally type it based on the value of another field like this.